### ISSUE TITLE
devrpm-el7 and devrpm-fedora smoke tests are failing 

### ISSUE THREAD

**Issue by Shwetha-Acharya** [gluster/project-infrastructure#187]
devrpm-el7 and devrpm-fedora smoke tests are failing  with the following error message:

15:30:41 rpmbuild --define '_topdir /home/jenkins/root/workspace/gh_devrpm-el7/extras/LinuxRPM/rpmbuild' -bs rpmbuild/SPECS/glusterfs.spec
15:30:42 Wrote: /home/jenkins/root/workspace/gh_devrpm-el7/extras/LinuxRPM/rpmbuild/SRPMS/glusterfs-2023.10.03-0.09.5959.el7.src.rpm
15:30:42 mv rpmbuild/SRPMS/* .
15:30:42 ERROR: Could not find required config file: /etc/mock/{build_flag}.cfg
15:30:42 ERROR: If you're trying to specify a path, include the .cfg extension, e.g. -r ./target.cfg

As these are the "Required" tests, merging of PRs are blocked and the release activity is affected.

@mscherer @rakshithakamath94

**Comment by mscherer** [comment]
@rakshithakamath94 looked at the issue, and we noticed that 1) the builder were reinstalled on the wrong version. Some change in ansible that triggered some variable priority change, and the automation dutifully executed.

Now, that was fixed, the problem seems to be on jenkins-update, as it doesn't replace {build_flag} in the script as before.

**Comment by mscherer** [comment]
I suspect it come from https://github.com/gluster/build-jobs/commit/891c0b70b385f25d3d2b293da2df7b3c2dae9ca7

**Comment by mscherer** [comment]
@rakshithakamath94 you remember why this was changed ? Is it ok if I revert the change partially ?

**Comment by rakshithakamath94** [comment]
Hi @mscherer ,

When I noticed that the job was failing I made some manual changes to see if I can fix them. You can revert the changes, sorry I forgot to revert them :D

**Comment by rakshithakamath94** [comment]
Actually when I saw the parameter {build_flag} not getting the value, I searched the job's definition to see if I can get it. I am glad that you fixed it.

**Comment by mscherer** [comment]
Mhh, but I fixed nothing, the job is still failling. Do you remember why the job use include-raw-escape ?

**Comment by rakshithakamath94** [comment]
I don't remember exactly. But it's introducing extra set of brackets now isn't it? As in, `${some_variable}` is now getting created as `${{some_variable}}` right

**Comment by mscherer** [comment]
Yup. I guess I can revert on that job, it can't break it more than now.

**Comment by mscherer** [comment]
So that change nothing. I suspect a change on jjb that trigger the problem, but maybe I didn't fix it right.

**Comment by mscherer** [comment]
Ok so I just forgot to merge the PR. I had to debug the whole build of jjb to figure out the issue.

**Comment by Shwetha-Acharya** [comment]
@mscherer is the issue fixed now? Please update when fixed, so that I can re trigger the smoke runs

**Comment by mscherer** [comment]
I think it sopuld work. Sorry, I didn't looked at my github notifications
